### DISCUSSION TITLE
player.isActive() problem

### DISCUSSION THREAD

**Discussion by gary7530** [pschatzmann/arduino-audio-tools#710] [Q&A] [answered]
Hello, when I use player.isActive(), an exception occurs. The song has already finished playing, but player.isActive() keeps returning true instead of false.

```
player.begin();
player.setVolume(1);
player.setAutoNext(false);
```

```
if (player.isActive() == true)
    Serial.println("we're active!");
else 
    Serial.println("trying to play...");
```

**Comment by pschatzmann** [comment, marked as answer]
isActive() returns true after you call begin() and false when you call end(). 
Alternatively you can halt/resume the processing by calling setActvie(bool) - or stop() and play() .

Using my latest commit you can check the number of bytes the copy() was processing...
